src/mono/wasm/debugger/debugger.md

```diff
@@ -0,0 +1,23 @@
+# Mono Wasm Debugger
+
+## Native debugging
```

Comment:
Isn't this document the same as https://paper.dropbox.com/doc/Blazor-Debugging-YQ5uISzLCRo9u2Oss8YTU? Can't we just copy-paste here? I like dropbox version more because it has photos, so it's easier to follow.

Reply:
Yes, done.